Update 0057-TRAN-transfers.md (#1420)
vegaprotocol/specs: protocol/0057-TRAN-transfers.md

  - [ ] In the first epoch no trading occurs and nothing is transferred to the reward account at the end of the epoch
  - [ ] In the second epoch, 2 * 10^6 trading occurs, and at the end of the epoch the transfer to the reward account occurs
  - [ ] At the end of the third epoch, no transfer occurs

If the network parameter <transfer.minTransferQuantumMultiple> is modified, this modification is applied

> immediately on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0059-TRAN-059" href="#0059-TRAN-059">0059-TRAN-059</a>)

Reviewer: ```suggestion
immediately on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0057-TRAN-058" href="#0057-TRAN-058">0057-TRAN-058</a>)
```

> immediately on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0057-TRAN-058" href="#0057-TRAN-058">0057-TRAN-058</a>)

Reviewer: ```suggestion
immediately on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0057-TRAN-061" href="#0057-TRAN-061">0057-TRAN-061</a>)
```

If the network parameter <transfer.minTransferFeeFactor> is modified, this modification is applied

> immediately, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0061-TRAN-061" href="#0061-TRAN-061">0061-TRAN-061</a>)

Reviewer: ```suggestion
immediately, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0057-TRAN-059" href="#0057-TRAN-059">0057-TRAN-059</a>)
```

> If the network parameter <spam.protection.maxUserTransfersPerEpoch> is modified, this modification is applied from the next Epoch on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0063-TRAN-063" href="#0063-TRAN-063">0063-TRAN-063</a>)

Reviewer: ```suggestion
If the network parameter <spam.protection.maxUserTransfersPerEpoch> is modified, this modification is applied from the next Epoch on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0057-TRAN-060" href="#0057-TRAN-060">0057-TRAN-060</a>)
```

  - [ ] In the first epoch no trading occurs and nothing is transferred to the reward account at the end of the epoch
  - [ ] In the second epoch, 2 * 10^6 trading occurs, and at the end of the epoch the transfer to the reward account occurs
  - [ ] At the end of the third epoch, no transfer occurs

If the network parameter <transfer.minTransferQuantumMultiple> is modified, this modification is applied

> from the next block on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0059-TRAN-059" href="#0059-TRAN-059">0059-TRAN-059</a>)

Reviewer: ```suggestion
immediately on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0059-TRAN-059" href="#0059-TRAN-059">0059-TRAN-059</a>)
```

> If the network parameter <transfer.minTransferFeeFactor> is modified, this modification is communicated to the frontend clients and applied there appropriately (displayed/rejected on the frontend if the available funds are insufficient) (<a name="0060-TRAN-060" href="#0060-TRAN-060">0060-TRAN-060</a>)

Reviewer: ```suggestion
```

If the network parameter <transfer.minTransferFeeFactor> is modified, this modification is applied

> from the next block on, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0061-TRAN-061" href="#0061-TRAN-061">0061-TRAN-061</a>)

Reviewer: ```suggestion
immediately, i.e., transfers are accepted/rejected according to the new parameter. This holds for both increase and decrease. (<a name="0061-TRAN-061" href="#0061-TRAN-061">0061-TRAN-061</a>)
```

> If the network parameter <spam.protection.maxUserTransfersPerEpoch> is modified, this modification is communicated to the frontend clients and applied there appropriately at the start of the next epoch (displayed/rejected on the frontend if too many transfers are send within the epoch) (<a name="0062-TRAN-062" href="#0062-TRAN-062">0062-TRAN-062</a>)

Reviewer: ```suggestion
```

  - [ ] In the first epoch no trading occurs and nothing is transferred to the reward account at the end of the epoch
  - [ ] In the second epoch, 2 * 10^6 trading occurs, and at the end of the epoch the transfer to the reward account occurs
  - [ ] At the end of the third epoch, no transfer occurs

> If the network parameter <transfer.minTransferQuantumMultiple> is modified, this modification is communicated to the frontend clients and applied there appropriately (displayed/too small transfers rejected). (<a name="0058-TRAN-058" href="#0058-TRAN-58">0058-TRAN-058</a>)

Reviewer 2: Im happy we can test these ACs, the statement about modification is communicated to the frontend clients - Should we see a trx rejection message returned back from wallet/core/tx_error event to convey the error. Also, should be have a FE scenario to cover ensuring transfer rejections are conveyed correctly on the FE?

Reviewer: I'd argue this is badly worded. The value of the network parameter is visible to FE via the usual APIs - they can deal with it as they do with all the other things. 

We may wish to have "push" notifications for FE for various things where it would be equally important - we don't. It would be a completely separate discussion IMO.  

I suggest we just delete this AC.

> If the network parameter <transfer.minTransferQuantumMultiple> is modified, this modification is communicated to the frontend clients and applied there appropriately (displayed/too small transfers rejected). (<a name="0058-TRAN-058" href="#0058-TRAN-58">0058-TRAN-058</a>)

Reviewer: ```suggestion
```
